### DISCUSSION TITLE
Libsass removal and @import deprecation

### DISCUSSION THREAD

**Discussion by samuel-clara** [bdkjones/CodeKit#719] [General]
I'm wondering which approach do you suggest for maintaining older projects that relies on Codekit and Libsass; and most important the ones with a ton of stylesheets bind together with @import, as this will not be supported by future versions of Dart Sass? will the only solution be to update each old project manually (or of course to not update Codekit) or will Codekit provide some backward compatibility (for example an option to choose the Dart Sass version)? the bigger deal for me is definitely the removal of @import support

**Comment by bdkjones** [comment]
I agree that removing `@import` is going to be a massive pain for old codebases.

Maintaining multiple versions of Dart Sass in CodeKit isn't really viable because the API changes between versions occasionally. Your best bet is to invest some time modernizing old codebases to `@use` as it comes up. The farther you fall behind the current state of Sass, the higher the technical debt you'll have to pay down at some future point.

**Reply by samuel-clara** [reply]
This will be a nightmare 🙈💪
